examples/webgpu_multiple_rendertargets_readback.html

```diff
@@ -175,6 +174,8 @@
 
 				const selection = options.selection;
 
+				if ( pixelBuffer !== undefined ) renderer.recycleBuffer( pixelBuffer );
```

Comment:
This could be executed automatically in `readRenderTargetPixelsAsync`? It would be nice if we could hide `renderer.recycleBuffer()` from users.

Reply:
We don't know what the lifetime of the returned buffer is after it is in the hands of the developer, unless you mean they could pass it in as a parameter to `readRenderTargetPixelAsync( ..., bufferFromLastCall )`?

Reply:
Maybe if o the user use `RenderTarget.dispose()` or reuse `readRenderTargetPixelAsync()`?